source/php/Component/Acceptance/acceptance.blade.php

```diff
@@ -0,0 +1,45 @@
+<!-- acceptance.blade.php -->
+<div id="{{ $id }}" class="{{ $class }} js-suppressed-iframe" {!! $attribute !!}>
```

Comment:
If .js-suppressed-iframe isn't in classList it won't be able to be altered or removed via filters in WP. Do we want this to be possible or not @sebastianthulin?

Reply:
Moved to classList